### ISSUE TITLE
Autocomplete pop-up steals the focus when composing requests, leading to a buggy experience

### ISSUE THREAD

**Issue by raxityo** [ProxymanApp/Proxyman#1375]
(Apologies if this has been reported. I was not able to find this when I searched for issues on GitHub)

## Description:
When composing a request in "Edit and Repeat" window, the autocomplete pop-up of headers steals the focus while typing, so it's very hard to type the header keys or values if there's a custom value to be written.

## Steps to Reproduce:
1. Make a request to https://httpbin.org/post
2. In Proxyman, edit and repeat the request
3. Try to add a header "User-Agent" and any value for that header
4. Observe how the autocomplete pop-up  steals the focus

## Current Behavior:
https://user-images.githubusercontent.com/767683/193056962-2ee23b58-0987-456b-8914-c786663d1300.mp4

## Expected Behavior:

I would expect the autocomplete pop-up to not steal the focus

## Environment:

- App version: Version 3.10.0 (31000)
- macOS version: macOS Monterey 12.6

**Comment by raxityo** [comment]
Yes, it pops up as soon as I start typing something in the header key or value field that would match one of the known headers.

To get back to the focused area, I would have to click within the field I was editing. Hitting `Esc` key hides the pop-up, but still loses the focus

**Comment by NghiaTranUIT** [comment]
Thanks. I'm able to reproduce it if I enable the "Use Keyboard navigation" on the Keyboard -> Shortcut

To workaround, you can disable it.

![Screen_Shot_2022-09-29_at_22_09_21](https://user-images.githubusercontent.com/5878421/193069453-5dfb64bd-aa82-4d13-866d-451711b46cd0.jpg)

**Comment by NghiaTranUIT** [comment]
@raxityo Fixed build here: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_3.10.0_Fix_the_AutoCompletion_issue_on_Compose_and_Breakpoint.dmg

Technically, I improve the behavior of the Auto-Completion Popover, it doesn't steal the focus anymore. Work with " use Keyboard Navigation to move Focus between controls" 👍
